### ISSUE TITLE
Adding a Surface Normal View to the type of views available

### ISSUE THREAD

**Issue by Mayankm96** [Microsoft/AirSim#249]
I saw in UnrealCV that it allows an additional view to have surface normals of the objects seen by the camera. 

I am wondering how to incorporate the surface normal view of UnrealCV into AirSim so that I can record that view from it along with the RGB view (which I suppose I can do from the APIs given). 

I would appreciate help on this matter. Thank you!

**Comment by Mayankm96** [comment]
I tried installing the unrealCV plugin to my project. I couldn't see if it working or not because the rendering became really slow and it was really difficult to type on the UE terminal. 

Any ideas anyone?

**Comment by Mayankm96** [comment]
So I tried making my own material for getting normals from the UE4Editor. The design is somewhat similar to what has been done in UnrealCV. However, I am now struggling to get the plugin working. I went through the `Source` folder of the AirSim plugin and added my own variables for rendering the view whereever I saw the other views being mentioned. But as expected I don't think it simply works like that. 

![normal](https://cloud.githubusercontent.com/assets/12863862/26740813/f24f9fe4-478b-11e7-944e-be400d14a719.PNG)
